source/basic.tex

```diff
@@ -4074,9 +4074,11 @@
 \item
 a \defn{pointer past the end of} an object\iref{expr.add}, or
 \item
-the \defn{null pointer value}\iref{conv.ptr} for that type, or
+\indextext{null pointer value|see{value, null pointer}}
+the \defnx{null pointer value}{value!null pointer}\iref{conv.ptr} for that type, or
```

Comment:
Can you check *.ilg for warnings, just to be sure?

Reply:
$ grep warnings *.ilg | grep -v " 0 warnings"
<empty>

Can't find any warnings.